password_reminder.php

```diff
@@ -133,7 +133,7 @@
 		if (!$msg->containsErrors()) 
 		{
 			//save data
-			$password   = $_POST['form_password_hidden'];
+			$password   = $usersDAO->is_sha1($_POST['form_password_hidden']);
```

Comment:
When is_sha1() returns `false`, an error should be reported rather than proceeding with the database update.